examples/mdx/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const base = require('../../webpack.config.base');
+
+// Customized babel loader with the minimum we need to get `mdx` libraries
+// working, which unfortunately codegen JSX instead of JS.
+const babelLoader = {
+  loader: require.resolve('babel-loader')
```

Comment:
We need a `devDependencies` for this (technically in root, but should have here if we're not getting config from root).